packages/fuchsia_ctl/bin/main.dart

```diff
@@ -129,11 +136,10 @@ Future<OperationResult> ssh(
       identityFilePath: identityFile,
     );
   }
-  final OperationResult result = await sshClient.runCommand(
-    targetIp,
-    identityFilePath: identityFile,
-    command: args['command'].split(' '),
-  );
+  final OperationResult result = await sshClient.runCommand(targetIp,
+      identityFilePath: identityFile,
+      command: args['command'].split(' '),
+      timeoutMs: int.parse(args['timeout-seconds']) * 1000);
```

Comment:
This parameter should just be a `Duration`.

Reply:
Done